frontend/www/js/omegaup/components/qualitynomination/Details.vue

```diff
@@ -15,7 +15,7 @@
         </div>
         <div class="row">
           <div class="col-sm-3">
-            <strong>{{ T.wordsNominator }}</strong>
+            <strong>{{ T.wordsNominatedBy }}</strong>
```

Comment:
Estamos tratando el dejar de usar el sufijo `word` en las cadenas de traducción porque carece de contexto, se podrá modificar por:
```suggestion
            <strong>{{ T.qualityNominationNominatedBy }}</strong>
```
?